### PR TITLE
Add kibana 8.13 image

### PR DESCRIPTION
**Pull request by jamonation**
### Image Size

- [x] The Image is smaller in size than its common public counterpart.
- [ ] The Image is larger in size than its common public counterpart (please explain in the notes).

Notes:

### Image Vulnerabilities

- [ ] The Grype vulnerability scan returned 0 CVE(s).
- [x] The Grype vulnerability scan returned > 0 CVE(s) (please explain in the notes).

Notes:

### Image Tagging

- [ ] The image is _not_ tagged with version tags.
- [x] The image is tagged with :latest
- [ ] The image is not tagged with :latest (please explain in the notes).

Notes:

### Basic Testing - K8s cluster

- [ ] The container image was successfully loaded into a kind cluster.
- [ ] The container image could not be loaded into a kind cluster (please explain in the notes).

Notes:

### Basic Testing - Package/Application

- [x] The application is accessible to the user/cluster/etc. after start-up.
- [ ] The application is not accessible to the user/cluster/etc. after start-up. (please explain in the notes).

Notes:

### Helm

- [ ] A Helm chart has been provided and the container image can be used with the chart.  If needed, please add a -compat package to close any gaps with the public helm chart.
- [ ] A Helm chart has been provided and the container image is not working with the chart (please explain in the notes).
- [x] A Helm chart was not provided.

Notes:

### Processor Architectures

- [x] The image was built and tested for x86_64.
- [ ] The image could not be built for x86_64 (please explain in the notes).
- [x] The image was built and tested for aarch64.
- [ ] The image could not be built for aarch64. (please explain in the notes).

Notes:

### Functional Testing + Documentation

- [x] Functional tests have been included and the tests are passing.  All tests have been documnted in the notes section.

Notes:

### Environment Testing + Documentation

- [x] There has not been a request and/or there is no indication that this image needs tested on a public cloud provider.
- [ ] The container image has been tested successfully on a public cloud provider (AWS, GCP, Azure).
- [ ] The container image has not been tested successfully on a public cloud provider (AWS, GCP, Azure) (please explain in the notes).

Notes:

### Version

- [x] The package version is the latest version of the package.  The latest tag points to this version.
- [ ] The package version is the not the latest version of the package (please explain in the notes).

Notes:

### Dev Tag Availability

- [x] There is a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base')
- [ ] There is not a dev tag available that includes a shell and apk tools (by depending on 'wolfi-base') (please explain in the notes).

Notes:

### Access Control + Authentication

- [ ] The image runs as `nonroot` and GID/UID are set to 65532 or upstream default
- [x] Alternatively the username and GID/UID may be a commonly used one from the ecosystem e.g: postgres
- [ ] The image requires a non-standard username or non-standard GID/UID (please explain in the notes).

### ENTRYPOINT

- [x] applications/servers/utilities set to call main program with no arguments e.g. [redis-server]
- [ ] applications/servers/utilities not set to call main program with no arguments e.g. [redis-server] (please explain in the notes)
- [ ] base images leave empty.
- [ ] base image and not empty (please explain in the notes).
- [ ] dev variants is set to entrypoint script that falls back to system.
- [ ] dev variants is not set to entrypoint script that falls back to system (please explain in the notes).

### CMD

- [x] For server applications give arguments to start in daemon mode (may be empty)
- [ ] For utilities/tooling bring up help e.g. `–help`
- [ ] For base images with a shell, call it e.g. [/bin/sh]

### Environment Variables

- [ ] Environment variables added.
- [x] Environment variables not added and not required.

### SIGTERM

- [x] The image responds to SIGTERM (e.g., `docker kill $(docker run -d --rm cgr.dev/chainguard/nginx)`)

### Logs

- [x] Error logs write to stderr and normal logs to stdout. Logs DO NOT write to file.

### Documentation - README

- [ ] A README file has been provided and it follows the README template.

### CODE REVIEW

File: images/kibana/tests/main.tf [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  name      = "kibana"
+  inventory = data.imagetest_inventory.this
+
+  envs = {
+    IMAGE_NAME = var.digest
+    TAG        = local.parsed.pseudo_tag
+  }
+}
+
+resource "imagetest_feature" "this" {
```
Comment (mamccorm) [inline images/kibana/tests/main.tf:31]: Is it possible to increase the test coverage here and include some core functionality tests?

Comment (jamonation) [reply images/kibana/tests/main.tf:31]: I tried a few things, the best approximation that it is working is connecting up to an Elasticsearch. Without one available, Kibana just sits waiting to initialize. If Kibana detects an elasticsearch (by default it looks for 127.0.0.1:9200), it autoconfigures itself and spits out the `Kibana is now available` message.

I'll try to figure out a curl command to post something to it, or elasticsearch and validate with another test.

File: images/kibana/tests/main.tf [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+data "imagetest_inventory" "this" {}
+
+
+resource "imagetest_harness_docker" "this" {
+  name      = "kibana"
+  inventory = data.imagetest_inventory.this
+
+  envs = {
+    IMAGE_NAME = var.digest
+    TAG        = local.parsed.pseudo_tag
```
Comment (jamonation) [inline images/kibana/tests/main.tf:27]:
```suggestion
```

Comment (jamonation) [reply images/kibana/tests/main.tf:27]: Unused

File: images/kibana/tests/main.tf (hunk 1 above)
Comment (jamonation) [reply images/kibana/tests/main.tf:31]: Without elasticsearch logs look like this:

```
[2024-05-16T20:19:38.332+00:00][INFO ][http.server.Preboot] http server running at http://localhost:5601
[2024-05-16T20:19:38.375+00:00][INFO ][plugins-system.preboot] Setting up [1] plugins: [interactiveSetup]
[2024-05-16T20:19:38.381+00:00][INFO ][preboot] "interactiveSetup" plugin is holding setup: Validating Elasticsearch connection configuration…
[2024-05-16T20:19:38.392+00:00][INFO ][root] Holding setup until preboot stage is completed.


i Kibana has not been configured.

Go to http://localhost:5601/?code=574708 to get started.
```

Comment (mamccorm) [reply images/kibana/tests/main.tf:31]: could we plug in in our ECK operator I wonder?

Comment (jamonation) [reply images/kibana/tests/main.tf:31]: The operator looks unopinionated enough that I think that might just work @mamccorm 🎉  didn't even realize we had it. Will give it a try and see what it does

File: images/kibana/tests/main.tf [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+  values = {
+    image = {
+      repository = "cgr.dev/chainguard/eck-operator"
+      tag        = "latest"
+    }
+  }
+}
+
+resource "imagetest_feature" "my-awesome-k3s-test" {
```
Comment (jamonation) [inline images/kibana/tests/main.tf:46]:
```suggestion
resource "imagetest_feature" "basic" {
```

File: images/kibana/tests/main.tf [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+data "imagetest_inventory" "this" {}
+
+resource "imagetest_harness_k3s" "this" {
+  name      = "eck-operator"
+  inventory = data.imagetest_inventory.this
+
+  sandbox = {
+    envs = {
+      IMAGE : "${local.parsed.registry_repo}:${local.parsed.pseudo_tag}"
```
Comment (jamonation) [inline images/kibana/tests/main.tf:25]:
```suggestion
      IMAGE : "${var.digest}"
```

**Review by mamccorm — APPROVED** [review]

**Pull request merged**